Wplace API.md

```diff
@@ -232,28 +262,39 @@ let countries = {ad: 'Andorra', ae: 'United Arab Emirates', ...}
+```
 
 ### Extra Colors
 
 ???
 
 ### Colors
 
-???
+0: Transparent
+1-31: Free Colors* (in order as shown in the place menu)
+32+: Extra Colors (in order as shown in the place menu)
```

Comment:
Markdown places all of these on a single line. Use `<br>` or add additional empty lines in between.